docs/developer/developing.md

```diff
@@ -0,0 +1,46 @@
+# Developing Cincinnati
+
+## Overview
+
+The following sections describe how to build and test the project for local development. 
+
+
+## Prerequisites
+
+* Git
+* Rust
+* [just](https://github.com/casey/just)
+
+*Note:* We recommend using [Rustup](https://github.com/rust-lang/rustup/blob/master/README.md) for Rust toolchain management. 
+
+## Running it locally
+
+Please below commands in parallel (for e.g. in different terminals at the same time).
```

Comment:
nit: missing a verb.  Maybe "Please run the following commands..."?  Or "Please run the commands below ..."?  Although I personally [don't feel a need to say "please"][1], since folks can follow these directions or not without it bothering me either way ;).

https://github.com/openshift/cincinnati/pull/198#discussion_r370885704

Reply:
Fixed in 1ed279c.